config.libsonnet

```diff
@@ -65,6 +65,7 @@
     certExpirationCriticalSeconds: 1 * 24 * 3600,
     cpuThrottlingPercent: 25,
     cpuThrottlingSelector: '',
+    kubeAPILatencyLimit: 1,
```

Comment:
Would you rename this to something like `kubeAPIWarningLatencySeconds` so it is more clear what it does?